sdk/python/orm/sqlalchemy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: SQLAlchemy
+sidebarTitle: SQLAlchemy
+description: Configure SQLAlchemy to work with your Turso database
+---
+
+![SQLAlchemy Quickstart](/images/guides/sqlalchemy-banner.png)
+
+## Prerequisites
+
+Before you start, make sure you:
+
+- [Install the Turso CLI](/cli/installation)
+- [Sign up or login to Turso](/cli/authentication#signup)
+
+<Steps>
+
+<Step title="Install the libSQL dialect for SQLAlchemy">
+
+```bash
+pip install sqlalchemy-libsql
+```
+
+</Step>
+
+<Step title="Configure database credentials">
```

Comment:
It might be worth [creating a snippet](https://mintlify.com/docs/reusable-snippets) we can include since this step is repeated.